src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -267,6 +274,14 @@ public Optional<Email> getParentEmail() {
             return Optional.ofNullable(parentEmail);
         }
 
+        public void setEducation(Education education) {
```

Comment:
Perhaps you could add javadocs comments for the methods written here for better readability